marquez_airflow/models.py

```diff
@@ -30,8 +30,26 @@ def __repr__(self):
                           {self.description!r},{self.ordinal_position!r})"
 
 
+class DbTableName:
+    def __init__(self, value: str):
+        parts = value.split('.')
+        if len(parts) > 2:
+            raise ValueError(
+                f"Table {value} must contain the schema and table name "
+                " separated by '.', or table name only."
+            )
+        self.value = parts[1] if len(parts) == 2 else parts[0]
+
+    def __eq__(self, other):
+        return self.value == other.value
+
+    def __repr__(self):
+        return f"DbTableName({self.value!r})"
```

Comment:
I would output the canonicalized version of the name here "{schema}.{name}" to simplify the code so that you don't have to turn lists of DBTableNames into list of strings?

Reply:
Make sense to provide a better string representation for the object in this case 👍